src/style.css

```diff
@@ -0,0 +1,118 @@
+html {
+  font-family: Roboto, sans-serif;
+  --color-main-text: #060b37;
+  --color-secondary-text: #616070;
+  --color-accent: #00acdc;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.product-card {
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 0 16px;
+}
+
+.product-card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.product-card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 150%;
+  color: var(--color-main-text);
+  margin: 0 16px 4px;
+}
+
+.product-card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 140%;
+  color: var(--color-secondary-text);
+  margin: 0 16px 16px;
+}
+
+.product-card__rating {
+  display: flex;
+  flex-direction: row-reverse;
+  margin: 0 16px 22px;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  height: 16px;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--active :nth-child(-n+4) {
+  background-image: url(/images/star-active.svg);
+  margin-right: 4px;
+}
```

Comment:
use all styles from `stars` task
for 3 active stars, 2 etc

Reply:
Why? On the mockup it is 4 active stars.